custom/conf/app.example.ini

```diff
@@ -617,7 +617,10 @@ ROUTER = console
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
 ;; The path of git executable. If empty, Gitea searches through the PATH environment.
-PATH =
+;PATH =
+;;
+;; The HOME directory for Git
+;HOME_PATH = %(APP_DATA_PATH)/home
```

Comment:
What about `%(APP_DATA_PATH)/git_home`? `home` feels too generic.

Reply:
That would be inconsistent with dasherized directories, IIRC. Maybe `git-home`?

Reply:
All fine by me. But we should specify its git's home.

Reply:
TBH, I am not sure whether it should be a general home. For example, make Gitea use it as its home in the future. Including SSH, etc.

Reply:
> TBH, I am not sure whether it should be a general home. For example, make Gitea use it as its home in the future. Including SSH, etc.
> 

Wouldn't that be `/data/git` from the container's perpective?

Reply:
There was no option for it, not managed by Gitea. That's why I gave up before and used the RepoRootPath.

Reply:
Oh. I just realize that due to customization within the helm chart that directory _appears_ to be dedicated for such content. But in the plain docker image it isn't. Nevermind.

Reply:
I think home is fine.